.travis.yml

```diff
@@ -8,11 +8,12 @@ node_js:
     - '0.12'
 notifications:
     irc: 'irc.freenode.org#tastejs'
-branches:
-    only:
-        - master
+addons:
+    sauce_connect: true
 env:
     global:
+        - secure: K1WJRK2WTcc8+ZVGfwgOR8dBSM3VqJXNzcFpRPbEd48kVikx6nKYhrSffyl8a5QM5+Z7UP1TrmRAmOwR09Qp7Lh2Ch7vE9ooK9gk3XsoUP454aWm35NzjqagyajzGe7disCf6nWxgPZ0lgNkCG18aBfyh9FlUryzGzRyiSqeVeQ=
+        - secure: Or0NreVqcSIE6lbmSe3TmeXlm/++hXZAN9LVozvwlRLrE2jevIYRWD6arep9RTACtiQOPLrLrMh1/KDlIMgSfdrSa/w72J+YchSrYm+d06p4BcHPZEDnooiktTx4e9nL/5MfxIUYNOF0iT+wReRxId0INukqIUvjKB4F/X7MgmE=
```

Comment:
People might wonder what is included in these. Might be worth it to include some comments similar to the ones on lines 16 till 27.